.drone.yml

```diff
@@ -450,7 +458,7 @@ volumes:
 ################################################
 
 kind: pipeline
-type: exec
+type: docker
```

Comment:
Why are we changing all the pipelines from `kubernetes` and `exec` to `docker`? What is the impact of this change?

Reply:
Discussed offline, this is temporary change for debugging and will be rolled back.